doc/source/whatsnew/v0.18.1.txt

```diff
@@ -29,6 +29,56 @@ New features
 Enhancements
 ~~~~~~~~~~~~
 
+Method chaininng improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Following methods / ``Indexer`` now accept ``callable`` as input. It is
+intended to make these more useful in the method chain.
+(:issue:`11485`, :issue:`12533`)
+
+- ``.where`` and ``.mask``
+- ``.query``
+- ``.loc``, ``iloc`` and ``.ix``
+
+When ``callable`` is passed, it is evaluated on the caller
+``Series/DataFrame`` prior to any other logics.
+
+.. note::
+
+   Thus, if input is callable ``np.ndarray`` or ``Series/DataFrame`` class
+   which defined on your own, it is evaluated on the caller
+   ``Series/DataFrame``, not regarded as values.
+
```

Comment:
make this a ``Furthermore`` and put ``.where/.mask`` after ``.query`` and indexers (make it a sub-section)